guides/common/modules/proc_acd_installation.adoc

```diff
@@ -13,36 +12,25 @@ This guide assumes you already have a working {BaseURL}Installing_Server_on_Red_
 endif::[]
 
 .Procedure
-ifdef::foreman-el,foreman-deb,katello,satellite[]
-. Create a new yum repository `/etc/yum.repos.d/foreman-plugins.repo` on your {ProjectServer}:
-+
-[options="nowrap", subs="verbatim,quotes,attributes"]
-----
-[foreman-plugins]
-name=Foreman plugins
-baseurl=https://yum.theforeman.org/plugins/{ProjectVersion}/el7/x86_64/
-enabled=1
-gpgcheck=0
-----
-endif::[]
-. Install the required packages on your {ProjectServer}:
-+
-[options="nowrap", subs="verbatim,quotes,attributes"]
-----
-# yum install tfm-rubygem-foreman_acd tfm-rubygem-smart_proxy_acd tfm-rubygem-smart_proxy_acd_core
-----
-. Run database migrations on your {ProjectServer}:
+. Install ACD on your {ProjectServer}:
 +
 [options="nowrap", subs="verbatim,quotes,attributes"]
 ----
-# foreman-rake db:migrate
-# foreman-rake db:seed
+# {foreman-installer} --enable-foreman-plugin-acd --enable-foreman-proxy-plugin-acd
```

Comment:
Perhaps this is better:
```suggestion
# {installer-scenario} --enable-foreman-plugin-acd --enable-foreman-proxy-plugin-acd
```

Then `This guide assumes you already have a working` becomes less relevant. You still need to have set up the repositories and installed the installer itself, there is no longer a need to have ran the installer before.

Reply:
I've changed/removed this. I think for a usage guide for a plugin, it's ok to assume users have a working Foreman installation.